Mackbellemore/soen-390-team07
# feat(client): search shipping location

#

## Changes this PR introduces

- Able to find shipping location dynamically (Google-like search bar) when creating a new shipment on shipping, order, and sales pages



## How to test this PR

- Can be tested using Netlify link
- Try to input any value and see if any component breaks



## ClickUp reference

CU-qx7zwx

## Notes for reviewers

- Clickup [link](https://app.clickup.com/t/qx7zwx)



## Checklist
